lib/menu_pizza.rb

```diff
@@ -0,0 +1,16 @@
+
+class Menu 
+
+  attr_accessor :dishes 
+
+  def initialize
+    @dishes = {
+      "cheese pizza"=>12.99, 
```

Comment:
I like the fact you've included several different menus to represent different restaurants